### PR TITLE
[PLATFORM-152] Pagination support in instamojo-py wrapper

### PR DESCRIPTION
**Pull request by sarthakk97**

### CODE REVIEW

File: instamojo_wrapper/api.py [hunk 1]
```diff
@@ -29,8 +29,25 @@ def auth(self, username, password):
...
+        query_dict = dict(
+            limit=limit,
+            page=page,
+        )
+
+        query_string = urlencode(dict((k, v) for k, v in query_dict.items() if v is not None))
+        if query_string:
+            path += '?' + query_string
+
+        print(path)
```
Comment (vijithv) [inline instamojo_wrapper/api.py]: can we remove this?

File: instamojo_wrapper/api.py [hunk 2]
```diff
@@ -497,6 +539,17 @@ def refunds_list(self):
...
         """
+        path = 'refunds/'
 
-        response = self._api_call(method='get', path='refunds/')
+        query_dict = dict(
+            limit=limit,
+            page=page,
+        )
+
+        query_string = urlencode(dict((k, v) for k, v in query_dict.items() if v is not None))
```
Comment (vijithv) [inline instamojo_wrapper/api.py]: can we move the 'adding query param' logic to separate method?

File: instamojo_wrapper/api.py [hunk 3]
```diff
@@ -497,6 +539,17 @@ def refunds_list(self):
...
+        if query_string:
+            path += '?' + query_string
+
+        print(path)
```
Comment (vijithv) [inline instamojo_wrapper/api.py]: Please remove all print statements

File: instamojo_wrapper/api.py (hunk 1 above)
Comment (sarthakk97) [reply instamojo_wrapper/api.py]: The print statement, right?
I'll do it asap

**Pull request merged**